azurelinuxagent/common/logger.py

```diff
@@ -186,7 +186,7 @@ class AppenderType(object):
     TELEMETRY = 3
 
 
-def add_logger_appender(appender_type, level=LogLevel.INFO, path=None):
+def add_logger_appender(appender_type, level=LogLevel.INFO, path=None, ):
```

Comment:
Very curious - what does just adding a `,` to a function do?

Reply:
Sorry. I think its a typo. I don't think it should be like this.

Reply:
of course the next question is why did the tests did not catch it? :)

Reply:
Will change this code.

Just for info: Python allows a "," at the end (it treats the call_args as a tuple)
```
>>> def hello(yo,):
...     print(yo)
... 
>>> hello("Ping")
Ping
>>> hello("Ping", "Ting")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: hello() takes exactly 1 argument (2 given)
```
Also more discussion [here](https://stackoverflow.com/a/17492103).

Reply:
Honestly had no idea about this. The more you know!